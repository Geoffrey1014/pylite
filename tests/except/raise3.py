def main():
    raise "String"

print "Start"
main()
print "End"
