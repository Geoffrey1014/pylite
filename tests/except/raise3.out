Start
TypeError:
  At raise3.py, line 2, in main()
  At raise3.py, line 5, in __init__()
